user-interface/src/case-assignment/CaseAssignmentScreen.test.tsx

```diff
@@ -164,36 +162,32 @@ describe('CaseAssignment Component Tests', () => {
   });
 
   test('/cases should contain table displaying assigned cases when all cases are assigned', async () => {
-    vi.spyOn(Chapter15MockApi, 'list').mockImplementation(
-      (_path: string): Promise<ResponseData> => {
-        return Promise.resolve({
-          message: 'not found',
-          count: 0,
-          body: {
-            caseList: [
-              {
-                caseId: '081-23-44463',
-                caseTitle: 'Flo Esterly and Neas Van Sampson',
-                dateFiled: '2023-05-04',
-                assignments: ['Sara', 'Bob'],
-              },
-              {
-                caseId: '081-23-44462',
-                caseTitle: 'Bridget Maldonado',
-                dateFiled: '2023-04-14',
-                assignments: ['Frank', 'Sue'],
-              },
-              {
-                caseId: '081-23-44461',
-                caseTitle: 'Talia Torres and Tylor Stevenson',
-                dateFiled: '2023-04-04',
-                assignments: ['Joe', 'Sam'],
-              },
-            ],
+    vi.spyOn(Chapter15MockApi, 'get').mockImplementation((_path: string): Promise<ResponseData> => {
+      return Promise.resolve({
+        message: 'not found',
+        count: 0,
+        body: [
```

Comment:
Replace these with MockData.getCaseBasic() and assignments need to be removed as they are now lazy loaded.